.babelrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
You can remove this file because this is configured in webpack

Reply:
We did not